tests/by-util/test_cp.rs

```diff
@@ -1496,14 +1496,29 @@ fn test_copy_through_just_created_symlink() {
 }
 
 #[test]
-fn test_copy_through_dangling_symlink() {
+fn test_copy_through_dangling() {
     let (at, mut ucmd) = at_and_ucmd!();
     at.touch("file");
     at.symlink_file("nonexistent", "target");
     ucmd.arg("file")
         .arg("target")
         .fails()
-        .stderr_only("cp: not writing through dangling symlink 'target'");
+        .stderr_only(
+        "cp: cannot stat 'dangle': No such file or directory (os error 2)",
+    );
+}
+
+#[test]
+fn test_copy_through_dangling_symlink_dereference() {
```

Comment:
Shouldn't the name here be "no_dereference"?

Reply:
yup, will fix